locales/en.js

```diff
@@ -0,0 +1,5 @@
+export default {
+    HomeHeaderRight: {
```

Comment:
need to determine naming convention, class component or what other options are there?